dashboard/app/models/lesson.rb

```diff
@@ -71,11 +71,18 @@ class Lesson < ApplicationRecord
     assessment_opportunities
   )
 
-  # A lesson has an absolute position and a relative position. The difference between the two is that relative_position
-  # numbers the lessons in order in two groups 1. lessons that are numbered on the script overview page (lockable false OR has_lesson_plan true)
-  # 2. lessons that are not numbered on the script overview page (lockable true AND has_lesson_plan false)
+  # A lesson has an absolute position and a relative position. The difference
```

Comment:
@Hamms you might already know this but this is the full explanation for why there can be two lessons with the same `relative_position`. I don't think we would ever translate anything that didn't have `has_lesson_plan` set to true but it might be worth querying for `!lockable || has_lesson_plan` in the query above